### PR TITLE
Add crud API

### PR DESCRIPTION
**Pull request by BossZou**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -37,4 +37,9 @@ $ pytest --cov=milvus/client --cov-report=html
 You may need a milvus server which is running when you run unit test. See more details on [Milvus server](https://github.com/milvus-io/milvus).
 
 
+## Update CONTRIBUTING.md
+
+Add issue tips into CONTRIBUTING.md, make sure all issue tips are sort by issue number in ascending order.
```
Comment (yamasite) [inline CONTRIBUTING.md]: Update CONTRIBUTING.md or CHANGELOG.md? 

make sure all issue tips are **sorted** by

File: milvus/client/check.py [hunk 2]
```diff
@@ -50,36 +51,44 @@ def is_legal_array(array):
...
 
+def is_legal_numpy_array(array):
+    return False if array is None or array.size == 0 else True
+
+
 def is_legal_records(value):
-    if not isinstance(value, list) or len(value) <= 0:
-        raise ParamError('A vector must be a non-empty, 2-dimensional array and '
-                         'must contain only elements with the float data type.')
+    param_error = ParamError('A vector must be a non-empty, 2-dimensional array and '
```
Comment (yamasite) [inline milvus/client/check.py]: What about binary vectors? Should update this error message.

Comment (yamasite) [reply milvus/client/check.py]:
```python
param_error = ParamError('A vector must be a non-empty, 2-dimensional array and '
 'must contain only elements either with the float data type or the bytes data type.')
```

Comment (BossZou) [reply milvus/client/check.py]: solved.

**Pull request closed without merging**